app/views/contestants/index.html.erb

```diff
@@ -0,0 +1,5 @@
+<% @contestants.each do |cont| %>
+  <h3><%= cont.name %></h3>
+  <p>Projects: <%= cont.projects.pluck(:name).join(", ") %></p>
```

Comment:
We don't want to do query logic here in the view. The better choice here is to make a method in your model. So something like:
```ruby
def projects_list
  projects.pluck(:name).to_sentence (or you can use your .join)
end
```
then you can just call that method on your `cont`.